docs/adr/0002-videos-languages.rst

```diff
@@ -0,0 +1,64 @@
+ADR 0002 - Videos languages
+===========================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We want to think Marsha as accessible from the beginning. At least from the point of view of the videos, which are the main content available.
+
+We can think about a video as a "set of related videos with their accessibility counterparts".
+
+Counterparts
+~~~~~~~~~~~~
+
+Audio
++++++
+
+The same video can be proposed with different audio tracks. Which often result in many videos. That's our case, we don't manage audio tracks separately.
```

Comment:
actually we should be able to do both: several video (in the video alternative has signs for example) and audio tracks (for multilingual audio on the same video).

I think the term "audio tracks" must mean "several audio track files associated to one video file" otherwise it is confusing.

Reply:
I don't see why audio track and sign are related.

Someone not deaf but having difficulties to hear may want to pick a video with audio AND see, on the side, the video with the signs. 

Because if I'm not wrong, this video with signs is expected to be displayed on the side of the main video. Which is not the case for videos with different audio tracks. So for me, expect if I misunderstood something, both are not related at all (in term of modelization)

Reply:
Yes I agree, they are not related!
We need both but they are not related.
so we need:
- many audio tracks on the video
- many auxiliary video tracks (one per sign or other associated video content...)

Reply:
Ok so let's be more precise about audio tracks.

If I have a video, but want the audio in three languages. Do we store, at the end 3 videos, with track *included* in the video, Or a video, with audio stored separately (audio files) ? Or do we allow both ?

Reply:
Maybe like this:
```
class Video()
    file = File()  # the main video file

class VideoTrack()
    video = FK(Video)
    file = File()  # the associated video file for signs
 
class AudioTrack()
    video = FK(Video)
    file = File()  # the associated audio file for sound track
    lang = Char()

class SubtitleTrack()
    video = FK(Video)
    file = File()  # the associated srt or vtt file with subtitles
    lang = Char()
```
Where:

- video-tracks, audio-tracks and subtitle-tracks must all be synchronized with the main video.
- subtitle-tracks must be included from day-one.
- audio-tracks is for top of backlog
video-tracks if for bottom of backlog

Reply:
So to answer my questions: we manage audio language of a video as a separate audio file. No more audio included in the original video. Can we enforce this? If not, how can this be managed in a player?

Reply:
My understanding is that in the main video file there is an audio track. The audio tracks we link are just overrides that are proposed in the player.